clang/test/Sema/builtin-stdc-rotate.c

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -fsyntax-only -verify %s
```

Comment:
Should the target be pinned? Then the `#ifdef`s wouldn't be necessary anymore I think.